builders/integrations/indexers/subsquid.md

```diff
@@ -88,12 +94,55 @@ To get started indexing Substrate data on Moonbeam, you'll need to create a Subs
         const processor = new SubstrateBatchProcessor();
         processor.setDataSource({
           chain: '{{ networks.moonbase.rpc_url }}',
-          // Resolves to 'https://moonbase.archive.subsquid.io'
-          archive: lookupArchive('moonbase', { type: 'Substrate' }),
-        });
+          // Resolves to 'https://v2.archive.subsquid.io/network/moonbase-testnet'
+          archive: lookupArchive('moonbase', {type: 'Substrate', release: 'ArrowSquid'}),
+        })
+
         ```
 
-And that's all you have to do to configure your Subsquid project to index Substrate data on Moonbeam! Now you can update the `schema.graphql`, `typgen.json`, and `src/processor.ts` files to index the data you need for your project!
+5. There's one more quick change to make to the template. The Subsquid Substrate template is configured to process Substrate account types, but Moonbeam uses Ethereum-style accounts. In the `getTransferEvents` function, remove the ss58 encoding of the `from` and `to` fields. In an unmodified Substrate template, the `from` and `to` fields are ss58 encoded as shown:
+
+    ```ts
+    from: ss58.codec('kusama').encode(rec.from),
+    to: ss58.codec('kusama').encode(rec.to),
+    ```
+
+    After removing the ss58 encoding, the respective lines are:
+
+    ```ts
+    from: rec.from, 
+    to: rec.to, 
+    ```
+
+6. Launch Postgres and detach by running:
```

Comment:
I'm conflicted about having these steps here just because this guide was meant to be a quick start like this is all you need to change to get going on Moonbeam guide. I'm not opposed to having these steps here, but now I feel like we need screenshots for the terminal output for these steps and it's going to make this guide longer than I would have liked for it to be

Maybe add a separate section with these steps, like "Run Your Substrate or EVM Indexer" so at least these steps aren't duplicated